### PR TITLE
Events, Auth modules

### PR DESCRIPTION
**Pull request by ritiksr25**
- General
    - updated error handler code
    - updated logs
    - iife
    - minor changes
    - add scheduler for sending credentials
- Auth 
    - added send creds by email using kue
- Events 
    - added schemas for participants, events, attendance
    - participants funtionalities

### CODE REVIEW

File: controllers/events_controller.js [hunk 1]
```diff
@@ -0,0 +1,268 @@
...
+		kue.scheduleJob(args);
+		sendSuccess(res, participant);
+	}
+};
+
+module.exports.updateParticipant = async (req, res) => {
+	let { name, email, branch, year, phone } = req.body;
+	let updateObj = {
+		name,
+		email,
```
Comment (rhnmht30) [inline controllers/events_controller.js]: Should we allow updating of email and phone?
What if someone updates their email to something that is already being used?

Comment (ritiksr25) [reply controllers/events_controller.js]: no

Comment (rhnmht30) [reply controllers/events_controller.js]: Another scenario: What if a user enters a wrong email/phone by mistake and wants to update it later?

Comment (ritiksr25) [reply controllers/events_controller.js]: we will update directly in db

Comment (rhnmht30) [reply controllers/events_controller.js]: Let's think of some way to verify the email when someone updates it.

**Pull request merged**